about.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="icon"
+      type="image/*"
+      href="./assets/images/titleicon.png"
+      alt="Erriecha"
+    />
+    <link rel="stylesheet" href="./assets/main.css" />
+    <link rel="stylesheet" href="./assets/about.css" />
+    <title>Erriecha Annual | Festival 2023</title>
+  </head>
+  <body>
+    <header class="header">
+      <nav class="navbar-advert desktop-only align-section">
+        <ul class="order-advert">
+          <li class="list-advert">
+            <a class="link-advert">
+              <img
+                src="./assets/images/facebook.png"
+                class="image-advert"
+                alt="facebook"
+              />
+            </a>
+          </li>
+          <li class="list-advert">
+            <a class="link-advert">
+              <img
+                src="./assets/images/twitter.png"
+                class="image-advert"
+                alt="twitter"
+              />
+            </a>
+          </li>
+          <li class="list-advert">
+            <a class="link-advert"> English </a>
+          </li>
+          <li class="list-advert">
+            <a class="link-advert"> Mypage </a>
+          </li>
+          <li class="list-advert">
+            <a class="link-advert"> Logout </a>
+          </li>
+        </ul>
+      </nav>
+      <nav class="navbar-primary desktop-only align-section">
+        <ul class="order-logo">
+          <li class="list-logo">
+            <a class="link-logo" href="./index.html">
+              <img
+                src="./assets/images/titleicon.png"
+                class="image-logo"
+                alt="Erriecha Logo"
+              />
+            </a>
+          </li>
+          <li class="list-logo list-close" id="popup-closeid">
+            <a href="#" class="link-logo close-icon">&#215;</a>
+          </li>
+        </ul>
+        <ul class="order-primary">
+          <li class="list-primary">
+            <a href="./about.html" class="link-primary link-about">About</a>
```

Comment:
- [ ] Since we are already on the About page, it would be great if this link takes us to the Home page. Kindly implement it for a better user experience. 🧐️